.github/workflows/build.yml

```diff
@@ -0,0 +1,15 @@
+name: build
+
+on:
+  release:
+    types: [ released ]
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build:
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish_pure_python.yml@v1
+    with:
+      upload_to_pypi: ${{ (github.event_name == 'release') && (github.event.action == 'released') }}
```

Comment:
Is this logic needed since the `on` above is confined to the same? Couldn't it just be `true`?

Reply:
well if a pull request (or manual dispatch) triggers the build, this wouldn't upload (only upon release)

Reply:
Got it. I glazed over the other two triggers.